UKHomeOffice/sgmr-service
# SPCR-69 Add functionality to delete a vessel

## Description
Add ability to be able to remove/delete vessels that have been saved previously to keep an up to date list of saved vessels.

When clicking on a vessel to edit it, there should be a button to delete the vessel. When this is clicked a page should appear asking if you are sure. When yes is clicked, the vessel is deleted and the vessel page will have a notification banner at the top with a success message.

### To test
- Go to the pleasure craft page
- Click on a saved vessel and scroll to the bottom
- Click the delete button
- This should take you to a page asking if you are sure.
- Click yes and then continue
- You should be redirected to the pleasure craft page with a success banner at the top
- Click another tab, and back to pleasure crafts
- This banner should no longer be visible


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
